spec/active_force/active_query_spec.rb

```diff
@@ -577,4 +571,18 @@ def check_ranges(base_query, start, finish, &format_block)
       expect(active_query.first.id).to eq("0000000000AAAAABBB")
     end
   end
+
+  describe "#ids" do
+    before do
+      allow(client).to receive(:query).and_return(api_result)
+      api_result.each do |instance|
+        allow(active_query).to receive(:build).with(instance, {}).and_return(build_restforce_sobject(id: instance['Id']))
```

Comment:
Our `sobject` double doesn't respond to `[]=` so we're building the AQs with actual `SObject`s so we're correctly testing behavior.